### ISSUE TITLE
Bug Report: yggdrasilctl getPeers fails

### ISSUE THREAD

**Issue by jasikpark** [yggdrasil-network/yggdrasil-go#417]
# Steps to reproduce

1. Install via the .pkg installer from https://yggdrasil-network.github.io/builds.html
2. Follow the FAQ and run `yggdrasilctl getPeers` expecting myself and a list of other peers to be printed.
3. Get error: 

```
2019/05/13 00:45:54 Found platform default config file /etc/yggdrasil.conf
2019/05/13 00:45:54 Using endpoint unix:///var/run/yggdrasil.sock from AdminListen
2019/05/13 00:45:54 Connecting to UNIX socket /var/run/yggdrasil.sock
2019/05/13 00:45:54 Fatal error: dial unix /var/run/yggdrasil.sock: connect: permission denied
```

# Versions

`macOS 10.14.4`
`yggdrasil 0.3.5` - https://880-115685026-gh.circle-artifacts.com/0/yggdrasil-0.3.5-macos-amd64.pkg

**Comment by Mikaela** [comment]
Is your user allowed to access `/var/run/yggdrasil.sock` ? Does it work with sudo?

**Comment by jasikpark** [comment]
that seems to be the problem. oh cool are you Mikaela from chat:privacytools.io? i'm @dan:privacytools.io on there - saw you promoting yggdrasil and finally decided to try it out

**Comment by jasikpark** [comment]
how would i fix that?

**Comment by Arceliar** [comment]
I would assume that `chmod` would work, as should running `sudo yggdrasilctl`. I think the correct thing to do, ideally, is to run `yggdrasil` as an unprivileged user (`setcap` the binary as needed, to let it create/use the tun/tap) and then add your user account to the same group, but I don't think the packages are set up to do that, and I'm too lazy to do it myself on my own machines, so honestly I just `sudo` everything 99% of the time...

**Comment by jasikpark** [comment]
yeah.... i'm just going to make myself use `sudo` all the time rather than actually figure it out. Someday maybe I'll make a PR for the mac installer and then I'll actually do it and try to make that the default.

**Comment by neilalexander** [comment]
Part of the problem is that macOS doesn't have `setcap` in the way that we do on Linux, so on macOS, Yggdrasil runs as root. The benefit it provides though is that unprivileged users can't `yggdrasilctl` to drop peers or change anything else, so there's that.
